src/app/Scenes/MyCollection/Screens/ArtworkForm/Components/ArtistAutosuggest.tsx

```diff
@@ -32,31 +51,98 @@ export const ArtistAutosuggest: React.FC<ArtistAutosuggestProps> = ({
             <AutosuggestResults
               query={artistQuery}
+              prependResults={filteredCollecteArtists}
               entities={["ARTIST"]}
               showResultType={false}
               showQuickNavigationButtons={false}
               onResultPress={onResultPress}
-              ListEmptyComponent={() => (
-                <Flex width="100%">
-                  <Text>We couldn't find any results for "{artistQuery}"</Text>
-                  {!!enableArtworksFromNonArtsyArtists && (
-                    <Flex alignItems="center" width="100%">
-                      <Button variant="outline" onPress={onSkipPress} mt={3} block>
-                        Can't find the Artist? Skip ahead
-                      </Button>
+              ListHeaderComponent={() =>
+                enableArtworksFromNonArtsyArtists ? (
+                  <Touchable
+                    onPress={onSkipPress}
+                    testID="my-collection-artwork-form-artist-skip-button"
+                  >
+                    <Flex flexDirection="row" my={1}>
+                      <Text variant="xs" color="black60">
+                        Or skip ahead to{" "}
+                      </Text>
+                      <Text variant="xs" color="black60" underline>
+                        add artwork details
+                      </Text>
                     </Flex>
-                  )}
-                </Flex>
-              )}
+                  </Touchable>
+                ) : null
+              }
+              ListEmptyComponent={() =>
+                enableArtworksFromNonArtsyArtists ? (
+                  <Flex width="100%" my={2}>
+                    <Text>We didn't find "{artistQuery}" on Artsy.</Text>
+                    <Text>You can add their name in the artwork details.</Text>
+                    <Button variant="outline" onPress={onSkipPress} mt={4} block>
+                      Add Artist
+                    </Button>
+                  </Flex>
+                ) : (
+                  <Flex width="100%">
+                    <Text>We couldn't find any results for "{artistQuery}"</Text>
+                  </Flex>
+                )
+              }
             />
-          </Flex>
+          </Box>
         ) : null}
       </Box>
     </SearchContext.Provider>
   )
 }
+
+const ArtistAutosuggestScreenQuery = graphql`
+  query ArtistAutosuggestQuery {
+    me {
+      myCollectionInfo {
+        collectedArtistsConnection(first: 100) {
+          edges {
+            node {
+              __typename
+              displayLabel
+              formattedNationalityAndBirthday
+              imageUrl
+              initials
+              internalID
+              isPersonalArtist
+              slug
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+export const filterArtistsByKeyword = (
+  artists: Array<{ displayLabel: string | null }>,
+  keywordFilter: string
+) => {
+  if (keywordFilter?.length < 2) {
```

Comment:
small suggestion (nitpick): no need for optional chaining since keywordFilter is string
```suggestion
  if (keywordFilter.length < 2) {
```

Reply:
Good catch! Fixed it 👍